Assets/BossRoom/Scripts/Client/Net/ClientGameNetPortal.cs

```diff
@@ -211,6 +219,51 @@ public static bool StartClientRelayMode(GameNetPortal portal, string roomKey, ou
             return true;
         }
 
+        public static async void StartClientUnityRelayModeAsync(GameNetPortal portal, string joinCode)
+        {
+
+            var chosenTransport  = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().UnityRelayTransport;
+            NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
+
+            switch (chosenTransport)
+            {
+                case UnityTransport utp:
+                    Debug.Log($"Setting Unity Relay client with join code {joinCode}");
+
+                    await UnityServices.InitializeAsync();

```

Comment:
You mention in another comment this isn't used, not sure I understand. If it's not useful, why call it?
If it is, is there a "if (!IsInitialized)" we could do, just like you did for signin?
It just feels wrong to initialize UnityServices each time we click the "Connect" button. Maybe it's an idempotent call and I'm worrying for nothing?

Reply:
The code that wasn't used was removed. And for the UnitySerivces.Initialize is fine to be called multiple times at runtime